checks/container.go

```diff
@@ -40,27 +40,27 @@ func (c *ContainerCheck) Endpoint() string { return "/api/v1/container" }
 // RealTime indicates if this check only runs in real-time mode.
 func (c *ContainerCheck) RealTime() bool { return false }
 
-// Run runs the ContainerCheck to collect a list of running containers and the
+// Run runs the ContainerCheck to collect a list of running ctrList and the
 // stats for each container.
 func (c *ContainerCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.MessageBody, error) {
 	start := time.Now()
-	containers, err := container.GetContainers()
+	ctrList, err := util.GetContainers()
 	if err != nil {
 		return nil, err
 	}
 
 	// End check early if this is our first run.
-	if c.lastContainers == nil {
-		c.lastContainers = containers
+	if c.lastRates == nil {
+		c.lastRates = util.KeepContainerRateMetrics(ctrList)
```

Comment:
personal preference I think I'd call this `ExtractContainerRateMetric` which seems clearer to me that it will return something. `Keep` could imply some side-effect with no return.